lib/dashboard/monitor.js

```diff
@@ -1,42 +1,16 @@
 let blessed = require("neo-blessed");
-let CommandHistory = require('./command_history.js');
+let LightMonitor = require('./light_monitor');
 
-class Dashboard {
+class Monitor extends LightMonitor {
```

Comment:
I renamed the class from Dashboard to Monitor
There is already a Dashboard file with Dashboard class which confused me at first.